### DISCUSSION TITLE
nebula一个tag中的属性能否存储多个相同类型的值

### DISCUSSION THREAD

**Discussion by techSunRise** [vesoft-inc/nebula#5237] [Q&A]
nebula一个tag中的属性能否存储多个相同类型的值
例如：一个点有school这个tag，该tag含有一个name属性，如何使用nebula将北京邮电大学、北邮这两个不同的值存储在info这个tag的name属性中

**Comment by QingZ11** [comment]
🤔 ，北京邮电和北邮不是一所学校么。你的需求是不是搜索 `name == 北邮` 的时候，把北京邮电大学也搜出来？
